foo/lib/config/configloader.py

```diff
@@ -0,0 +1,98 @@
+import os
+from typing import Optional
+
+import boto3
+import yaml
+
+
+class ConfigLoaderException(Exception):
+    pass
+
+
+class Config:
+
+    def __init__(self, config: dict, source: str):
+        self._config = config
+        self._source = source
+
+    @property
+    def source(self):
+        return self._source
+
+    def __repr__(self):
+        return (f'Config(config={self._config!r}'
```

Comment:
It would fit nicely in one line and would be more representative of end results, also those spaces from line below will be visible (IDK if intended, looks bad anyway).